fastlane/Fastfile

```diff
@@ -458,15 +469,61 @@ platform :ios do
   #####################################################################################
   desc 'Downloads localized metadata for App Store Connect from GlotPress'
   lane :download_localized_strings_and_metadata do |options|
-    # WordPress: leverage the default release toolkit setup.
-    # It's not clear in the action name, but this downloads both `.strings`
-    # translations for the app and App Store Connect metadata.
-    ios_update_metadata(options)
-    # Jetpack: use the custom setup. Notice that we don't need extra steps to
-    # download `.strings` because Jetpack uses the same codebase as WordPress.
+    # Download `Localizable.strings` translations used within the app
+    parent_dir_for_lprojs = File.join(PROJECT_ROOT_FOLDER, 'WordPress', 'Resources')
+    ios_download_strings_files_from_glotpress(
+      project_url: 'https://translate.wordpress.org/projects/apps/ios/dev/', # TODO: Use a constant to DRY stuff
```

Comment:
This is also used in the lane that checks translation converage, so there's some DRY-ing to be done here